sim/build.sbt

```diff
@@ -1,6 +1,5 @@
 import Tests._
 
-// This needs to stay in sync with the chisel3 and firrtl git submodules
 val chiselVersion = "3.4.1"
```

Comment:
Similar to CY's comment. Can we confirm what Chisel version is really used in the final jar?

Reply:
3.4.1